models/src/main/scala/coop/rchain/models/BitSetBytesMapper.scala

```diff
@@ -0,0 +1,40 @@
+package coop.rchain.models
+
+import com.trueaccord.scalapb.TypeMapper
+
+import scala.collection.immutable.BitSet
+import java.nio.ByteBuffer
+
+import com.google.protobuf.ByteString
+
+object BitSetBytesMapper {
+  val BYTES_PER_LONG = 8
+
+  implicit val bitSetBytesMapper: TypeMapper[ByteString, BitSet] =
+    TypeMapper(
+      byteStringToBitSet
+    )(bitSetToByteString)
+
+  def bitSetToByteString(bitset: BitSet): ByteString = {
+    val longs: Array[Long] = bitset.toBitMask
+    val bytes: Array[Byte] = (ByteBuffer.allocate(BYTES_PER_LONG * longs.length) /: longs) {
+      (acc, long) =>
+        acc putLong long
```

Comment:
Please make sure that this is picking a specific endianness.

Reply:
Done as little endian as discussed in Discord.